Ports/bash/patches/0001-accept.c-Include-sys-select.h.patch

```diff
@@ -10,10 +10,10 @@ serenity is not one of them.
  1 file changed, 1 insertion(+)
```

Comment:
Commit message nitpick: The commit title doesn't convey what we are doing. Something like "Ports: Remove getconf from bash examples" would be better.